### ISSUE TITLE
Squeezenet1_1 got fail_accuracy

### ISSUE THREAD

**Issue by mengfei25** [intel/torch-xpu-ops#501]
### 🐛 Describe the bug

torchbench_float16_training
xpu  train squeezenet1_1                      
E0626 12:44:17.969000 140227443300160 torch/_dynamo/utils.py:1478] RMSE (res-fp64): 0.00654, (ref-fp64): 0.00153 and shape=torch.Size([4, 1000]). res.dtype: torch.float16, multiplier: 2.000000, tol: 0.001000
fail_accuracy

loading model: 0it [00:00, ?it/s]

Loading pipeline components...:   0%|          | 0/6 [00:00<?, ?it/s][A

Loading pipeline components...:  33%|███▎      | 2/6 [00:00<00:00, 17.23it/s][A

Loading pipeline components...:  67%|██████▋   | 4/6 [00:00<00:00,  7.04it/s][A

Loading pipeline components...: 100%|██████████| 6/6 [00:01<00:00,  4.18it/s][A
Loading pipeline components...: 100%|██████████| 6/6 [00:01<00:00,  4.88it/s]

loading model: 0it [00:05, ?it/s]

### Versions

torch-xpu-ops: https://github.com/intel/torch-xpu-ops/commit/31c400195d63064940242220dc9100322d36bac4
pytorch: 0f81473d7b4a1bf09246410712df22541be7caf3 + PRs: 127277,129120
device: PVC 1100, 803.61, 0.5.1

**Comment by retonym** [comment]
similar issue in https://github.com/intel/torch-xpu-ops/issues/507, add fp16 datatype label in that issue
